### ISSUE TITLE
[Feature Addition] Medical Image Scanning feature

### ISSUE THREAD

**Issue by Snehal-Reddy** [Snehal-Reddy/MedAI#6]
Implement a medical image scanning feature, like breast cancer tumors.

**Comment by kraviral** [comment]
I want to work on this issue.

**Comment by apprenticearnab** [comment]
I also want to work on this issue

**Comment by apprenticearnab** [comment]
Can I use tensorflow and keras for implementing this?

**Comment by Snehal-Reddy** [comment]
Sure ! Join the slack channel (http://bit.ly/2LCeuEW) and we can continue the discussion there.

**Comment by apprenticearnab** [comment]
Please tell me , should I only implement if the tumor is benign or malignant or implement the feature of masking the area of the image affected by tumor effectively based on the image provided ?

**Comment by Snehal-Reddy** [comment]
Hey ! Please join and slack channel (http://bit.ly/2LCeuEW) and we will discuss there along with other members working on this issue.
